openfl-tutorials/interactive_api/Tensorflow_MNIST/new_python_api_Tensorflow_MNIST.ipynb

```diff
@@ -435,7 +435,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Can we discard these changes? Since it looks like you've just run the example, and info about this run was saved.

Reply:
Sure, thanks, I've removed it.